tests/test_convert_to_onnx.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import unittest
+import itertools
+from parameterized import parameterized
+
+import onnx
+import torch
+
+from monai.networks import convert_to_onnx
+from monai.networks.nets import UNet
+
+
+TESTS = list(itertools.product(["cpu", "cuda"], [True, False], ["CUDAExecutionProvider", "CPUExecutionProvider"]))
+
+class TestConvertToOnnx(unittest.TestCase):
+    @parameterized.expand(TESTS)
+    def test_value(self, device, use_ort=False, ort_provider=None):
+        model = UNet(
```

Comment:
`UNet` should be compatible with ONNX but not all models will be so we'll have to have per-model tests like we do with Torchscript. This can be left to future PRs if it's not a priority however.

Reply:
I added test case for SegResNet. More test cases shall be added in subsequent PRs.

Reply:
We can test the networks like we do for Torchscript, that is testing with random data on small versions of the networks since we're not concerned with results. Testing the network as they are in core suffices for testing the bundles that use them, you don't need to worry about testing bundles directly.

Reply:
Thanks @ericspod , I created test_onnx_save test utility and applied it to TestRetinaNet. It does no cross validation between PyTorch and onnxruntime.

Reply:
Hi @liqunfu ,
     Why do you add a `test_onnx_save` and apply it to TestRetinaNet. I think the `test_convert_to_onnx` and `test_bundle_onnx_export` is enough for this PR. We may add the bundle onnx test in MONAI model-zoo to test more models. Could you please remove these tests, if there are no other reasons?

Thanks,
Bin

Reply:
As @ericspod commented, we need a way to test MONAI's nets with onnx in the same way nets are tested for TorchScript. For this purpose, RetinaNet is tested in test_retinanet via test_onnx (similar to test_script) by calling test_onnx_save. Eventually, all test_xxxnet.py (or similar) will have a test_onnx call to validate the net's definition with onnx export. It is for validation of individual torch nnmodules in MONAI.

On the other hand, test_convert_to_onnx is to test convert_to_onnx utility in case it is broken.

test_bundle_onnx_export (thank to you for adding it!) is to ensure the utility added to script (onnx_export) is working as we expected. so that we can use it to generate model bundles.